config/rbac/README.md

```diff
@@ -0,0 +1,3 @@
+# Note about RBAC
+
+We manually manage RBAC-related configuration directly in the Helm chart.
```

Comment:
_thought_: should we provide a pointer to `helm-chart/dash0-operator/templates/operator/cluster-roles.yaml` here?

Reply:
link added :link:

Reply:
Well the actual permissions are really mostly listed in cluster-roles.yaml only, I think. But just linking to the chart is good, I agree 👍